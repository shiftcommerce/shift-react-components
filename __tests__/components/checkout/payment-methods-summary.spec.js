// Libraries
import React from 'react'

// Components
import PaymentMethodsSummary from '../../../src/components/checkout/payment-methods-summary'

test("renders the shipping methods summary", () => {
  // Arrange
  const paymentMethod = 'credit-card'

  // Act
  const wrapper = shallow(<PaymentMethodsSummary paymentMethod={paymentMethod} title={'test'}/>)

  // Assert
  expect(wrapper).toMatchSnapshot()
  expect(wrapper.find('div').first()).toMatchSelector('.c-payment-methods__summary')
  expect(wrapper.find('div').last()).toMatchSelector('.c-payment-methods__summary-information')
  expect(wrapper).toIncludeText(paymentMethod)
})
