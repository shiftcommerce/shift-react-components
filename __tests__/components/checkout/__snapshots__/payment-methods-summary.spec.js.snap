// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the shipping methods summary 1`] = `
<div
  className="c-payment-methods__summary"
>
  <PaymentMethodHeader
    collapsed={true}
    title="test"
  />
  <div
    className="c-payment-methods__summary-information"
  >
    <p>
      credit-card
    </p>
  </div>
</div>
`;
